collectors/apicmcpfault.py

```diff
@@ -0,0 +1,58 @@
+import logging, re
+
+from prometheus_client.core import CounterMetricFamily, Summary
+import BaseCollector
+
+LOG = logging.getLogger('apic_exporter.exporter')
+REQUEST_TIME = Summary('network_apic_mcp_fault_counter',
+                       'Time spent processing request')
+
+
+class ApicIPsCollector(BaseCollector.BaseCollector):
```

Comment:
This name is taken https://github.com/sapcc/apic-exporter/blob/4c26f98c531e2e87b49341b9b02c4a1436566fc5/collectors/apicips.py#L11
Maybe you could name it "ApicMCPCollector"?

Reply:
updated.